src/Admin/templates/globals/partials/head.html.twig

```diff
@@ -0,0 +1,42 @@
+<head>
+    <title>{{ title|translate }}</title>
```

Comment:
Do note that this kind of translating (from a variable) will make it impossible for us to find the term that needs translating in the translation pipeline.

Reply:
Good catch